backend/app/api/v1/endpoints/ocp/ocpJobs.py

```diff
@@ -12,7 +12,7 @@
 @router.get(
     "/api/v1/ocp/jobs",
     summary="Returns a job list",
-    description="Returns a list of jobs in the specified dates. \
+    description="Returns a list of jobs in the specified dates of requested size. \
```

Comment:
Again, "of requested size" is potentially confusing, and it's also incomplete as it doesn't mention the offset and so forth. I'd just stick with "in the specified date range".

Reply:
Carried out in PR#135

Reply:
Well, now that's merged and it's still here.